# Request
---
# Response
bool setreset_para_accepted
